express-api/src/typeorm/Migrations/1728680019519-AddAdminAreaToERPNotifications.ts

```diff
@@ -0,0 +1,50 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class AddAdminAreaToERPNotifications1728680019519 implements MigrationInterface {
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(`
+            -- update the "New Disposal Project Submitted template" first because of different formatting and spacing
+            -- add a new line for the Site Location using the admin area name
+            UPDATE notification_template
+            SET body = REGEXP_REPLACE(
+                          body, 
+                          'Site Address: {{ property.Address1 }}' || chr(10) || '[[:space:]]+<br>',  -- to find all instances of spaces before the <br> that follows after a new line
+                          'Site Address: {{ property.Address1 }}<br>' || chr(10) || chr(9) || chr(9) || chr(9) || chr(9) || chr(9) || 'Site Location: {{ property.AdministrativeArea.Name }}<br>',
+                          'g')  -- 'g' to replace all instances within a template
+            WHERE body LIKE '%Site Address: {{ property.Address1 }}' || chr(10) || '%<br>%' AND id = 1`);
+    await queryRunner.query(`
```

Comment:
There didn't seem to be a way to do a global search and replace based on the differences in formatting in template id=1, so had to split it up into 2 separate queries.